src/app/patient-summary/components/generics/resource.types.ts

```diff
@@ -38,10 +42,11 @@ type TRowTable<T> = {
 
 export type TRow<T> = TRowRow<T> | TRowTable<T>;
```

Comment:
I think `TRowRow` and `TRowTable` are confusing names. 
Maybe `TRowConfigRow` and `TRowConfigTable` would be clearer?

Reply:
Yes it's much clearer